ksql-engine/src/main/java/io/confluent/ksql/KsqlEngine.java

```diff
@@ -450,6 +456,8 @@ private PreparedStatement buildSingleDdlStatement(
       return new PreparedStatement(statementString, statement);
     } else if (statement instanceof SetProperty) {
       return new PreparedStatement(statementString, statement);
+    } else if (statement instanceof UnsetProperty) {
```

Comment:
Could just rolls this into the `SetProperty` if branch:

```
} else if (statement instanceof SetProperty || statement instanceof UnsetProperty) {
   return new PreparedStatement(statementString, statement);
}
```

Better still, switch this class to have a static map<? extends Statement, Handler> as we've done for `StandaloneExecutor`. This is a valid refactor given you're increasing the complexity of this method.  Or do it in a follow up PR.

Reply:
Ah, @hjafarpour, you're funny. The idea was that you did this in this PR or a follow up PR straight after, because you're adding complexity to an already-over-sized function. Yet, I'm guessing from your response that you have no intention to refactor this.